package.json

```diff
@@ -308,24 +302,24 @@
     "lodash": "4.17.21",
     "lodash-move": "1.1.1",
     "lodash-webpack-plugin": "0.11.5",
-    "luxon": "1.22.0",
```

Comment:
We had problems with this in the past, weird resolution at compilation. That's why we needed to lift it here. Probably yarn problems. We should keep an eye on this one if we remove it.

Reply:
I have a better idea now about how that could be a problem than I did when I removed it. We can add it back.